docs/odoo/models/ir.cron.rst

```diff
@@ -0,0 +1,152 @@
+
+
+**1. Creating the model and method of this model.**
+
+
+.. code-block:: shell
+
+        class model_name(models.Model):
+            _name = "model.name"
+            # fields
+            def method_name(self, cr, uid, context=None): # method of this model
+                # your code
+
+**2. Creating the automated action**
+
+
+    `If you want to build new modules in the guidelines from Odoo you should add the code for an automated action under yourDefaultModule/data/ in a separate XML file.`
+
+An important thing to note with automated actions is that they should always be defined within a noupdate field since this shouldn’t be updated when you update your module.
+
+.. code-block:: shell
+
+    <openerp>
+        <data noupdate="1">
+            <record id="unique_name" model="ir.cron">
+            	<field name="name">Name </field>
+            	<field name="active" eval="True" />
+            	<field name="user_id" ref="base.user_root" />
+            	<field name="interval_number">1</field>
+            	<field name="interval_type">days</field>
+            	<field name="numbercall">-1</field>
+            	<field name="doal">1</field>
+            	<!--<field name="nextcall" ></field>--> 
+            	<field name="model" eval="'model.name '" />
+            	<field name="function" eval="'method_name '" />
+            	<field name="args" eval="" />
+		<!--<field name="priority" eval="5" />--> 
+            </record>
+        </data>
+    </openerp>
+
+The first thing you’ll notice is the data ``noupdate="1"``, this is telling Odoo that all code within this tag shouldn’t be updated when you update your module.
+
+.. code-block:: shell
+
+    <record id="unique_name" model="ir.cron">
+
+The id is an unique identifier for Odoo to know what record is linked to which id. The model called ("ir.cron") is the model specifically made by Odoo for all automated actions. This model contains all automated actions and should always be specified.
+
+.. code-block:: shell
+
+    <field name="name">Name </field>
+
+The next line is the name. 
+
+.. code-block:: shell
+
+    <field name="active" eval="True" />
+
+Boolean value indicating whether the cron job is active or not.
+
+.. code-block:: shell
+
+    <field name="user_id" ref="base.user_root"/>
+
+This user id is referring to a specific user, in most cases this will be base.user_root.
+
+.. code-block:: shell
+
+    <field name="interval_number">1</field>
+
+Number of times the scheduler is to be called based on the "interval_type" 
+
+.. code-block:: shell
+
+    <field name="interval_type">days</field>
+Interval Unit.
+It should be one value for the list: ``'minutes'``, ``'hours'``, ``'days'``, ``'weeks'``, ``'months'``.
```

Comment:
Don't use quotes here. Correct format: ``minutes``

    It should be one value for the list: ``'minutes'``, ``'hours'``, ``'days'``, ``'weeks'``, ``'months'``.